bridge_adapters/src/lisp_adapters/string_char.rs

```diff
@@ -221,6 +220,16 @@ mod tests {
         val
     }
 
+    fn gensym(vm: &mut SloshVm, registers: &[Value]) -> VMResult<Value> {
```

Comment:
Why did this move to list_adapters?  This is a core lisp builtin so I don't see the logic here but maybe I am missing something.

Reply:
copy pasta sorry.